apdfl18/CPlusPlus/APDFL18.0.4PlusP2w/acrocolor_acrocolor.html

```diff
@@ -12,8 +12,8 @@
         <script async src="https://www.googletagmanager.com/gtag/js?id=G-L4MQFNK7VK"></script>
         <script>
             window.dataLayer = window.dataLayer || [];
-            function gtag(){dataLayer.push(arguments);}
-            gtag('js', new Date());  gtag('config', 'G-L4MQFNK7VK');
+��          function gtag(){dataLayer.push(arguments);}
+��          gtag('js', new Date());��gtag('config', 'G-L4MQFNK7VK');
```

Comment:
I'm seeing some goofy characters here - was this built on Linux?

Reply:
Windows

Reply:
rebuilt on linux